index.html

```diff
@@ -245,10 +245,10 @@ <h3>RailsGirls Summer of Code</h3>
 					<h3>Outreachy</h3>
 					<p>3-month internships for people traditionally underrepresented in tech. Interns work remotely with mentors from Free and Open Source Software communities.</p>
 				</a>
-				<a class="col-md-4" href="https://www.redhat.com/en/about/women-in-open-source" target="_blank" rel="noopener">
-					<img alt="" src="img/projects/womeninopensource.png" />
-					<h3>Women in Open Source Award</h3>
-					<p>We believe open source is the future of technology. It's time to recognize the contributions women are making and inspire a new generation to join the open source movement.</p>
+				<a class="col-md-4" href="https://gssoc.tech" target="_blank" rel="noopener">
+					<img alt="" src="img/projects/girlscriptsummerofcode.jpg" />
+					<h3>GirlScript Summer of Code</h3>
+					<p>3 month long Open Source program during summers, with an aim to help beginners get started with Open Source Development while encouraging diversity.</p>
```

Comment:
I think it's "during summer", not the plural form.

Reply:
It seems correct cause it’s recurring? (I mostly just copy the text from each project itself)